docs/DefiningLabels.md

```diff
@@ -0,0 +1,123 @@
+# Defining Labels
+
+Unless mentioned otherwise, **labels are applied to issues only** (not PRs). 
+
+## Label groups
+
+While GitHub does not have the concept of label groups, we can simulate label groups using systematic label naming. 
+i.e. labels having the same prefix are considered part of a label group.
+
+There are two types of label groups: 
+ 
+* Exclusive groups: No more than one label from the group can be applied to an issue
+* Non-exclusive groups:  Multiple labels from a group can be applied to an issue
+
+**Common label groups**:
+
+* exclusive: `s.` status, `p.` priority, `c.` category, `d.` difficulty, `e.` effort, 
+* non-exclusive:  `a-` aspect, `f-` feature, `t-` tech, 
+ 
+# Common Labels
+
+### Priority (`p.`)
+
+* `p.Critical`: Would like to fix it ASAP and release as a hot patch.
+* `p.Urgent`: Would like to handle in the very next release.
+* `p.High`: Enhances user experience significantly, would like to do in the next few releases.
+* `p.Medium`: Marginal impact on user experience.
+* `p.Low`: Very little impact, unlikely to do in the near future.
+* `p.Zero`: Unlikely to do, ever.
+
+> An issue is considered as _accepted_ when a priority label has been assigned.
+
+### Status (`s.`)
+
+* Open issues
+    * No status: New issue yet to be triaged. 
+    * `s.Ongoing` : The issue is being worked on.
+* Open PR  
+    * `s.ToReview`: Waiting for the review.
+    * `s.Ongoing` : The PR is being worked on.
+    * `s.ToMerge`: Main reviewer approved the changes. 
+    * `s.MergeApproved` : Both main reviewer and the code quality reviewer has approved the merge. PR can be merged.
+    * `s.OnHold` : The work on the PR has been put on hold pending some other event.
+* Closed issue/PR
+    * `s.Discarded` : The PR was abandoned. 
```

Comment:
This was introduced to easily identify discarded PRs through HubTurbo UI (Hubturbo doesn't show the merge status). Will remove.